### DISCUSSION TITLE
[Feature] Merge People page should show other persones sorted by similarity

### DISCUSSION THREAD

**Discussion by kingvuke** [immich-app/immich#5930] [Feature Request]
### The feature

In the Merge People page, it would be very helpful if the people were not sorted by number of photos, but if you had the option of sorting them by similarity.

This means that the system would have to compare the current person with all the others and sort them by distance.

### Platform

- [x] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4347
